metric_learn/base_metric.py

```diff
@@ -177,8 +214,57 @@ def transform(self, X):
                              accept_sparse=True)
     return X_checked.dot(self.transformer_.T)
 
-  def metric(self):
```

Comment:
Maybe we should keep this for now but mark it as deprecated, and point to the new `get_mahalanobis_matrix()` method.

Reply:
Yes, I agree, I forgot about that